AutoCollection/Performance.ts

```diff
@@ -147,7 +147,7 @@ class AutoCollectPerformance {
     private _trackMemory() {
         var totalMem = os.totalmem();
         var freeMem = os.freemem();
-        var usedMem = totalMem - freeMem;
+        var usedMem = process.memoryUsage().heapTotal;
         var percentUsedMem = usedMem / (totalMem || 1);
```

Comment:
It looks like a lot of these variables are no longer necessary (`totalMem`, `percentUsedMem`, `percentAvailableMem`). Could you remove them?